components/modal/index.jsx

```diff
@@ -79,11 +79,11 @@ const propTypes = {
 	 */
 	directional: PropTypes.bool,
 	/**
-	 * If true, Modals can be dismissed by clicking on the close icon or pressing esc key.
+	 * If true, Modals cannot be dismissed by clicking on the close icon or pressing esc key.
 	 */
-	dismissible: PropTypes.bool,
+	disableClose: PropTypes.bool,
 	/**
-	 * If true, Modals can be dismissed by clicking outside of modal. If unspecified, defaults to dismissible.
+	 * If true, Modals can be dismissed by clicking outside of modal. If unspecified, defaults to disableClose.
 	 */
 	dismissOnClickOutside: PropTypes.bool,
```

Comment:
We have another prop called `dismiss...`. I would think we would want to do the same to be consistent?

Reply:
hey @davidlygagnon, I didn't get you on this? can you kindly clarify :)

Reply:
I think what @davidlygagnon means is that we keep the semantics similar, e.g. change all `dismisss...` to `disable...`, similar to the change from `dismissable` to `disableClose`.

Reply:
@davidlygagnon What name would you recommend?

Reply:
Shouldn't that follow the same pattern as `disableClose`?
```
/**
 * If true, Modals can be dismissed by clicking outside of modal. If unspecified, defaults to disableClose.
 */
disableCloseOnClickOutside: PropTypes.bool,
```

Reply:
Got it. Yes, great suggestion.

`dismissOnClickOutside` is a default true also, it's just not in the default props, since it defers to the `dismissible` if undefined.

`dismissOnClickOutside` should become `disableCloseOnClickOutside` also.